iogt/templates/registration/password_reset_done.html

```diff
@@ -4,7 +4,7 @@
 <div class="profiles">
 	<p>{% trans "We've emailed you instructions for setting your password, if an account exists with the email you entered. You should receive them shortly." %}</p>
 	<p>{% trans "If you don't receive an email, please make sure you've entered the address you registered with, and check your spam folder." %}</p>
-	<a href="{% url 'home' %}" class="call-to-action__nav-item call-to-action__nav-item--search-home">
+	<a href="{% url 'home' %}" class="call-to-action__nav-item call-to-action__nav-item--back-home">
```

Comment:
@Mitso can you please change `{% url 'home' %}` to `{{request.site.root_page.url}}`